src/core/analytics.js

```diff
@@ -894,6 +918,26 @@ class Analytics {
       // convert integrations object to server identified names, kind of hack now!
       transformToServerNames(rudderElement.message.integrations);
 
```

Comment:
Can you add a comment with examples of these both structures & the expected output ?
 
 object.clientIntegrationObjects = object.successfullyLoadedIntegration; --> prepareDataForIntegrationsObj --> tempIntegrationsData
 const clientSuppliedIntegrations = rudderElement.message.integrations || { 'All' : true };

Reply:
@krishna2020 rather adding examples in file, we already have all the test cases [here](https://www.notion.so/rudderstacks/Test-Results-3f723a53a98f4f3183a48bfa086afa65), ideally there is a comment which is enough to explain what we are doing. we already have test cases through which we can check every scenarios

Reply:
It's not possible for any developer to search in notion page & find the test cases. A simple quick example here would help anyone checking the code.

Reply:
updated